Automattic/newspack-plugin
# Update WP GDPR Cookie Notice AMP handling for latest version of WP GDPR Cookie Notice

### All Submissions:






### Changes proposed in this Pull Request:



https://github.com/felixarntz/wp-gdpr-cookie-notice/pull/10 was merged and released in the latest version of WP GDPR Cookie Notice. That PR added a handy filter for turning off the endpoint, and that is much simpler than our current way of removing the ajax endpoint that fires on every page load from WP GDPR Cookie Notice.

The current implementation should continue working fine, but using the filter is less hacky and more robust.

### How to test the changes in this Pull Request:

1. In AMP mode with an incognito window and the network inspector open, verify when a cookie notice is shown, no network request is made:

<img width="1406" alt="Screen Shot 2020-08-24 at 12 08 30 PM" src="https://user-images.githubusercontent.com/7317227/91085955-0d42e680-e603-11ea-9164-f18917746f23.png">

### Other information:
